CryptoPrices/Sources/Home/Sources/Home/WalletStatisticSection.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  WalletStatisticSection..swift
+//  Home
+//
+//  Created by Minh Pham on 07/12/2022.
+//
+
+import Styleguide
+import SwiftUI
+
+public struct WalletStatisticSection: View {
+
+    public var body: some View {
+        ZStack {
+            gradientBackground
+            VStack(spacing: 40.0) {
+                totalCoins
+                todayProfit
+            }
+            .frame(
+                minWidth: 0,
```

Comment:
```suggestion
                minWidth: 0.0,
```

Reply:
oh I just check and actually we can clean up the `minWidth` attribute completely, let me update 💪